koku/api/metrics/views.py

```diff
@@ -36,23 +43,40 @@ class CostModelMetricsMapViewSet(mixins.ListModelMixin, viewsets.GenericViewSet)
 
     """
 
-    queryset = CostModelMetricsMap.objects.all()
     serializer_class = CostModelMetricMapSerializer
     permission_classes = (AllowAny,)
 
     def get_queryset(self):
-        """Get a queryset.
+        """
+        Get JSON from the cost_model_metrics_map.json.
 
-        Restricts the returned data to provider_uuid if supplied as a query parameter.
+        Filter on source_type
         """
-        queryset = CostModelMetricsMap.objects.all()
+        try:
+            with open(COST_MODEL_METRICS_FILE_NAME) as json_file:
+                cost_model_metrics_map = json.load(json_file)
```

Comment:
Can we consolidate the calls to load this JSON file?
Its done in 3 different places:
https://github.com/project-koku/koku/pull/1986/files#diff-999179f93d3cba58861a615b1899b2c4R303
https://github.com/project-koku/koku/pull/1986/files#diff-999179f93d3cba58861a615b1899b2c4R117

Preferably it would all go to one method and the data would be stored in a variable on the first load and then returned from the loaded variable on subsequent calls to that method.

Reply:
@chambridge I'll plan to store it in a constant instead of loading it from JSON.

Reply:
@chambridge see updated code. It's now a native python constant.